src/Controller/Back/SignalementEditController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Controller\Back;
+
+use App\Entity\Signalement;
+use App\Manager\SignalementManager;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/bo/signalements')]
+class SignalementEditController extends AbstractController
+{
+    #[Route('/{uuid}/edit-address', name: 'back_signalement_edit_address', methods: 'POST')]
+    public function validationResponseSignalement(
+        Signalement $signalement,
```

Comment:
C'est tout chaud, ça sort du four depuis la 6.3, tu peux créer un DTO et l'injecter à la place de l'objet de Request
https://symfony.com/blog/new-in-symfony-6-3-mapping-request-data-to-typed-objects